### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Vighnesh-V**
Open Source Issue Templates :)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (KeenWill) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Do we really need smartphone here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (KeenWill) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Is browser important? Or can we tailor this to be more ocaml-ly and compiler stuff
For example:
- OS
- camelot version
- ocaml version
- opam list
...

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Vighnesh-V) [reply .github/ISSUE_TEMPLATE/bug_report.md]: deleted !

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Vighnesh-V) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated!

**Pull request merged**